ds-collector-tests/cluster-dse-k8s.make

```diff
@@ -23,20 +23,30 @@ setup:
 	cp TEST-cluster-dse-k8s-*_secret.key collector/ || true
 	test -f collector/collect-info
 	# setup k8s cluster
-	wget https://thelastpickle.com/files/2021-01-31-cass_operator/01-kind-config.yaml -O /tmp/datastax/01-kind-config.yaml
+	cp k8s-manifests/01-kind-config.yaml /tmp/datastax/01-kind-config.yaml
```

Comment:
why the copy?

Reply:
Just following the previous pattern - why did not not use `-f` and the URL as elsewhere...was trying to limit figuring things out...